components/server/src/authorization/spicedb-authorizer.ts

```diff
@@ -26,10 +28,6 @@ export class SpiceDBAuthorizer {
             userId: string;
         },
     ): Promise<boolean> {
-        if (!this.client) {
-            return true;
```

Comment:
This was really confusing in testing :see_no_evil: 
And as we now require SpiceDB in every environment, we should remove it alltogether.